testing/tests/cpp_nesting.py

```diff
@@ -11,13 +11,28 @@
 
 from __future__ import unicode_literals
 
+import os
+import os.path as osp
+from textwrap import dedent
+
+import pytest
+
+from testing import TEST_PROJECTS_ROOT
 from testing.base import ExhaleTestCase
-from testing.decorators import confoverrides
-from testing.hierarchies import                                              \
-    class_hierarchy, clike, compare_class_hierarchy, compare_file_hierarchy, \
-    directory, file, file_hierarchy, namespace, union
+from testing.decorators import confoverrides, no_cleanup
```

Comment:
not sure if this was a copy-n-paste problem, but flake8 is having a problem with the unused import `no_cleanup`

```suggestion
from testing.decorators import confoverrides
```